test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -103,16 +105,23 @@ namespace iroha {
      */
     template <typename ExpectedQueryErrorType>
     void checkStatefulError(
-        QueryExecutorResult exec_result,
+        const QueryExecutorResult &exec_result,
         shared_model::interface::ErrorQueryResponse::ErrorCodeType
             expected_code) {
-      ASSERT_NO_THROW({
-        const auto &error_qry_rsp =
-            boost::get<const shared_model::interface::ErrorQueryResponse &>(
-                exec_result->get());
-        ASSERT_EQ(error_qry_rsp.errorCode(), expected_code);
-        boost::get<const ExpectedQueryErrorType &>(error_qry_rsp.get());
-      }) << exec_result->toString();
+      const shared_model::interface::ErrorQueryResponse *error_qry_rsp =
```

Comment:
I prefer not to shorten names like that.
In my view, there is no problem to name it like `error_query_response`. At least rsp sounds more like a cpu register :D

That is a minor issue.